### PR TITLE
[Camera] Update README.md

### PR DESCRIPTION
**Pull request by Snimm**
Slightly improved the documentation by listing the feature the camera plugin supports

*List which issues are fixed by this PR. You must list at least one issue.*

*If you had to change anything in the [flutter/tests] repo, include a link to the migration guide as per the [breaking change policy].*

## Pre-launch Checklist

- [x] I read the [Contributor Guide] and followed the process outlined there for submitting PRs.
- [x] I read the [Tree Hygiene] wiki page, which explains my responsibilities.
- [ ] I read and followed the [relevant style guides] and ran [the auto-formatter]. (Unlike the flutter/flutter repo, the flutter/plugins repo does use `dart format`.)
- [x] I signed the [CLA].
- [x] The title of the PR starts with the name of the plugin surrounded by square brackets, e.g. `[shared_preferences]`
- [ ] I listed at least one issue that this PR fixes in the description above.
- [ ] I updated `pubspec.yaml` with an appropriate new version according to the [pub versioning philosophy], or this PR is [exempt from version changes].
- [ ] I updated `CHANGELOG.md` to add a description of the change, [following repository CHANGELOG style].
- [x] I updated/added relevant documentation (doc comments with `///`).
- [ ] I added new tests to check the change I am making, or this PR is [test-exempt].
- [x] All existing and new tests are passing.

If you need help, consider asking for advice on the #hackers-new channel on [Discord].

[Contributor Guide]: https://github.com/flutter/plugins/blob/main/CONTRIBUTING.md
[Tree Hygiene]: https://github.com/flutter/flutter/wiki/Tree-hygiene
[relevant style guides]: https://github.com/flutter/plugins/blob/main/CONTRIBUTING.md#style
[CLA]: https://cla.developers.google.com/
[flutter/tests]: https://github.com/flutter/tests
[breaking change policy]: https://github.com/flutter/flutter/wiki/Tree-hygiene#handling-breaking-changes
[Discord]: https://github.com/flutter/flutter/wiki/Chat
[pub versioning philosophy]: https://dart.dev/tools/pub/versioning
[exempt from version changes]: https://github.com/flutter/flutter/wiki/Contributing-to-Plugins-and-Packages#version-and-changelog-updates
[following repository CHANGELOG style]: https://github.com/flutter/flutter/wiki/Contributing-to-Plugins-and-Packages#changelog-style
[the auto-formatter]: https://github.com/flutter/plugins/blob/main/script/tool/README.md#format-code
[test-exempt]: https://github.com/flutter/flutter/wiki/Tree-hygiene#tests

### CODE REVIEW

**Comment by Snimm** [comment]
> @Snimm are you still working on this? This needs a CLA and a [version bump](https://github.com/flutter/flutter/wiki/Contributing-to-Plugins-and-Packages#version-and-changelog-updates).

I did sign the cla.

File: packages/camera/camera/README.md [hunk 1]
```diff
@@ -16,6 +16,12 @@ A Flutter plugin for iOS, Android and Web allowing access to the device cameras.
 * Snapshots can be captured and saved to a file.
 * Record video.
 * Add access to the image stream from Dart.
+* Zoom camera
+* Flash support
+* Auto Focus
```
Comment (stuartmorgan) [inline packages/camera/camera/README.md:21]: Please be consistent about capitalization; this is using title case unlike all the other entries.

File: packages/camera/camera/README.md [hunk 2]
```diff
@@ -16,6 +16,12 @@ A Flutter plugin for iOS, Android and Web allowing access to the device cameras.
 * Snapshots can be captured and saved to a file.
 * Record video.
 * Add access to the image stream from Dart.
+* Zoom camera
```
Comment (stuartmorgan) [inline packages/camera/camera/README.md:19]: These all need periods for consistency.

File: packages/camera/camera/README.md [hunk 3]
```diff
@@ -16,6 +16,12 @@ A Flutter plugin for iOS, Android and Web allowing access to the device cameras.
...
+* Flash support
+* Auto Focus
+* ImageFormat selection for ImageStream and Video(iOS only).
```
Comment (stuartmorgan) [inline packages/camera/camera/README.md:22]: This is missing a space before (

**Review by stuartmorgan — CHANGES_REQUESTED** [review]

**Pull request closed without merging**